.circleci/config.yml

```diff
@@ -50,6 +50,19 @@ orbs:
                 ansible-playbook --syntax-check -e env=development dev.yml
                 ansible-playbook --syntax-check -e env=development server.yml
 
+      lint:
+        executor: python-3
+        steps:
+          - run: python --version
+          - checkout
+          # TODO: Cache pip OR install via apt-get.
+          - run: sudo pip install ansible-lint
+          - run: ansible-lint --version
+          - run: ansible-lint deploy.yml
+          - run: ansible-lint dev.yml
+          - run: ansible-lint server.yml
+          - run: ansible-lint rollback.yml
```

Comment:
Discussion:
- do we want to separate them into 4 different jobs so that we can get all errors at once?
- or, hack CircleCI like https://discuss.circleci.com/t/run-all-test-steps-even-if-one-fails/11229/9 ?
- or, keep it as it is?

Reply:
Forget me.
Using `when: always` to run all lint steps now.